applications/FluidDynamicsApplication/custom_constitutive/newtonian_two_fluid_2d_law.cpp

```diff
@@ -77,6 +78,13 @@ double NewtonianTwoFluid2DLaw::GetEffectiveViscosity(ConstitutiveLaw::Parameters
     double viscosity;
     EvaluateInPoint(viscosity, DYNAMIC_VISCOSITY, rParameters);
     const Properties& prop = rParameters.GetMaterialProperties();
+    const GeometryType &rGeom = rParameters.GetElementGeometry();
```

Comment:
```suggestion
    const auto& r_geom = rParameters.GetElementGeometry();
```
Note the `snake_case` typesetting for local variables. See the [style guide](https://github.com/KratosMultiphysics/Kratos/wiki/Style-Guide).